portal-api/src/v1/gef/controllers/validation/application.js

```diff
@@ -50,6 +50,7 @@ const validateApplicationReferences = (body = {}) => {
 
 const validatorStatusCheckEnums = (doc) => {
   const enumErrors = [];
+  // statuses received from TFM - shouldnt be updating to UKEF_ACKNOWLEDGED on its own
```

Comment:
Great stuff but it's in the wrong place - all of the statuses are OK for GEF, but Acknowledged is only received from TFM. Can we move this below so it's like

```js
case CONSTANTS.DEAL.GEF_STATUS.ABANDONED:
// intentional empty line here
// statuses received from TFM - GEF should not be updating to these itself
case CONSTANTS.DEAL.GEF_STATUS.UKEF_ACKNOWLEDGED:
```

Reply:
Expected a 'break' statement before 'case'.eslintno-fallthrough < --- I get this linting error when I move the comment down, should I add the eslint ignore where the empty line is?

Reply:
Let me know if the current changes are acceptable please